controllers/sriovoperatorconfig_controller.go

```diff
@@ -107,6 +108,8 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+	// Sort the policies with priority, higher priority ones is applied later
```

Comment:
nit: please add comment why this sorting is needed

Reply:
done